### PR TITLE
homework #2

### PR DESCRIPTION
**Pull request by sol4ik**

### CODE REVIEW

File: src/finance-content/finance-chart/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+    componentDidMount() {
+      const jsonFilePath = require("./content/" + this.props.tickerSymb + ".txt");
+        
+        fetch(jsonFilePath)
+          .then(response => {
+            return response.text()
+          })
+          .then(text => {
+            const chartData = JSON.parse(text);
```
Comment (pavlo-berezin) [inline src/finance-content/finance-chart/index.js:29]: Suggestion:
```
const { title, subtitle, data } = JSON.parse(text);

this.setState({ title, subtitle, data });
```

File: src/finance-content/finance-sidebar/finance-stats/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+// growth sign constants
+const up = '⬆';
+const down = '⬇';
+
+
+export default class FinanseStats extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            summaryData: {
```
Comment (pavlo-berezin) [inline src/finance-content/finance-sidebar/finance-stats/index.js:14]: Signs can be moved to some constant

File: src/finance-content/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import FinanceChart from './finance-chart';
+import FinanceSideBar from './finance-sidebar';
+
+import './finance-content.css';
+
+export default class FinanceContent extends React.Component {
+    render() {
+        return (
+            <div className="page-content full-width row-container">
+                    <FinanceChart tickerSymb={ this.props.tickerSymb }></FinanceChart>
```
Comment (pavlo-berezin) [inline src/finance-content/index.js:12]: fix pagination.

File: src/finance-page/finance-page.css [hunk 4]
```diff
@@ -1,19 +1,10 @@
...
-
-.page-content {
-    position: fixed;
-    top: 8em;
-
-    padding-bottom: 4em;
-}
-.row-container {
+    height: 100%;
+    min-height: 100vh;
```
Comment (pavlo-berezin) [inline src/finance-page/finance-page.css:3]: please use vh only on top level elements.

**Comment by pavlo-berezin** [comment]
it would be nice being able to get to pages that you've done without manually changing url.

File: src/finance-page/finance-page.css (hunk 4 above)
Comment (sol4ik) [reply src/finance-page/finance-page.css:3]: But technically finance-page is the top level, as App component has only font styles, and .root component has no styles as all.

**Pull request merged**